### PR TITLE
update slovenian translation:

### PR DESCRIPTION
**Pull request by gorazdko**
## Abstract

* status: ready, will finish the checklist in a separate PR 

### Checklist

- [x] first draft
- [ ] second draft
- [ ] ask a slovene colleague to review
- [ ] build and visual inspection of the translated GUI

#1582

### CODE REVIEW

File: translations/monero-core_sl.ts [hunk 1]
```diff
@@ -149,42 +149,42 @@
...
         <location filename="../pages/History.qml" line="161"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>Išči</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="191"/>
         <source>Date from</source>
-        <translation type="unfinished"></translation>
+        <translation>Datum od:</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: Please remove `:`

File: translations/monero-core_sl.ts [hunk 2]
```diff
@@ -149,42 +149,42 @@
...
     </message>
     <message>
         <location filename="../pages/History.qml" line="214"/>
         <source>Date to</source>
-        <translation type="unfinished"></translation>
+        <translation>Datum do:</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: remove `:`

File: translations/monero-core_sl.ts [hunk 3]
```diff
@@ -217,7 +217,7 @@
     <message>
         <location filename="../components/HistoryTable.qml" line="55"/>
         <source>Rings:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ringi</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: missing `:`

File: translations/monero-core_sl.ts [hunk 4]
```diff
@@ -303,7 +303,7 @@
     <message>
         <location filename="../components/HistoryTableMobile.qml" line="54"/>
         <source>Rings:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ringi</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: missing `:`

File: translations/monero-core_sl.ts [hunk 5]
```diff
@@ -791,7 +791,7 @@
     <message>
         <location filename="../components/PasswordDialog.qml" line="86"/>
         <source>Please enter wallet password for: </source>
-        <translation type="unfinished"></translation>
+        <translation>Vnesite geslo denarnice</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: missing `:` + whitespace

File: translations/monero-core_sl.ts [hunk 6]
```diff
@@ -827,12 +827,12 @@
...
         <location filename="../components/ProgressBar.qml" line="38"/>
         <source>%1 blocks remaining: </source>
-        <translation type="unfinished"></translation>
+        <translation>%1 preostalih blokov: </translation>
     </message>
     <message>
         <location filename="../components/ProgressBar.qml" line="70"/>
         <source>Synchronizing %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Sinhroniziram</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: Add missing place marker (`%1`)

File: translations/monero-core_sl.ts [hunk 7]
```diff
@@ -868,91 +868,91 @@
...
         <location filename="../pages/Receive.qml" line="373"/>
         <source>Create new address</source>
-        <translation type="unfinished"></translation>
+        <translation>Ustvari nov naslov</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="388"/>
         <source>Set the label of the new address:</source>
-        <translation type="unfinished"></translation>
+        <translation>Nastavi oznako za novi naslov</translation>
```
Comment (erciccione) [inline translations/monero-core_sl.ts]: missing `:`

**Comment by erciccione** [comment]
If you squash right now this can make it for 0.13

**Comment by gorazdko** [comment]
I squashed. Will finish the checklist with 0.13.1 though

**Comment by gorazdko** [comment]
alright, thank you

**Pull request merged**